openlibrary/plugins/openlibrary/api.py

```diff
@@ -308,8 +308,11 @@ def GET(self, _id):
             web.ctx.site.get('/books/%s' % _id)
             if re.match(r'OL[0-9]+M', _id)
             else models.Edition.from_isbn(_id)
-            
         )
+        if not edition:
+            return simplejson.dumps({
+                'error': 'Invalid ISBN 13.'
```

Comment:
What is the response code here? It looks like it will  be a 200 OK, it should be a HTTP 400 response.

Reply:
Yes the response should be a HTTP 400 response. Will ask @mekarpeles to test on dev and give feedback

Reply:
@hornc I have updated the pr to give 400 Bad Request error